src/main/java/edu/ucsb/nceas/mdq/MetadigContextListener.java

```diff
@@ -35,7 +35,7 @@ public void contextDestroyed(ServletContextEvent servletContextEvent) {
             try {
                 log.debug("Shutting down controller...");
                 controller.shutdown();
-                log.info("Controller shutdonw successfully.");
+                log.info("Controller shutdown successfully.");
```

Comment:
No need to change if you think too pedantic - but for next time: 
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/shut-down-shutdown